specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,560 @@
+        }
+      }
+    },
+    "ConversationIntent": {
+      "type": "object",
+      "description": "The intent classification result of a Conversation project.",
+      "required": [
+        "category",
+        "confidenceScore"
+      ],
+      "properties": {
+        "category": {
+          "description": "A predicted class.",
+          "type": "string"
+        },
+        "confidenceScore": {
+          "format": "float",
+          "x-ms-client-name": "confidence",
+          "description": "The confidence score of the class from 0.0 to 1.0.",
+          "type": "number",
+          "minimum": 0,
+          "maximum": 1
+        }
+      }
+    },
+    "ConversationEntity": {
+      "type": "object",
+      "description": "The entity extraction result of a Conversation project.",
+      "required": [
+        "category",
+        "text",
+        "offset",
+        "length",
+        "confidenceScore"
+      ],
+      "properties": {
+        "category": {
+          "description": "The entity category.",
+          "type": "string"
+        },
+        "text": {
+          "description": "The predicted entity text.",
+          "type": "string"
+        },
+        "offset": {
+          "format": "int32",
+          "description": "The starting index of this entity in the query.",
+          "type": "integer"
+        },
+        "length": {
+          "format": "int32",
+          "description": "The length of the text.",
+          "type": "integer"
+        },
+        "confidenceScore": {
+          "format": "float",
+          "x-ms-client-name": "confidence",
+          "description": "The entity confidence score.",
+          "type": "number"
+        },
+        "listKeys": {
+          "type": "array",
+          "description": "List of keys.",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "LUISTargetIntentResult": {
+      "type": "object",
+      "description": "It is a wrap up of LUIS Generally Available response.",
+      "x-ms-discriminator-value": "luis",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TargetIntentResult"
+        }
+      ],
+      "properties": {
+        "result": {
+          "type": "object",
+          "description": "The actual response from a LUIS Generally Available application.",
+          "x-nullable": true
+        }
+      }
+    },
+    "QuestionAnsweringTargetIntentResult": {
+      "type": "object",
+      "description": "It is a wrap up a Question Answering KB response.",
+      "x-ms-discriminator-value": "question_answering",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TargetIntentResult"
+        }
+      ],
+      "properties": {
+        "result": {
+          "type": "object",
+          "$ref": "../2021-07-15-preview/questionanswering.json#/definitions/KnowledgeBaseAnswers",
+          "description": "The generated answer by a Question Answering KB.",
+          "x-nullable": true
```

Comment:
Avoid the use of `x-nullable: true`. Properties with no value should simply be omitted from the payload.